apps/labs/posts/alt-text-scipy-2022.md

```diff
@@ -0,0 +1,72 @@
+---
+title: SciPy 2022 Accessibility Awareness Programs
+author: tony-fast
+published: July 12, 2022
+description: "Announcing the SciPy 2022 Accessibility Awareness Efforts"
+category: [Accessibility]
+# featuredImage:
+#   src: /posts/hello-world-post/featured.png
+#   alt: 'Excellent alt-text describing the featured image'
```

Comment:
Probably want to actually have alt text here?

First pass: "Close crop of a cartoon page showing a person writing alt text on an image that is part of a map, along with portions of text reading "ALT TEXT ... SCIENTIFIC DIAGRAMS ... DOCUMENTATION"

Reply:
these are commented out. there's alt text on the hero image. i didnt include a feature image.